Samplero/Samplero/Screen/ShopBasket/View/Cell/ShopBasketCollectionViewCell.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  ShopBasketCollectionViewCell.swift
+//  Samplero
+//
+//  Created by DaeSeong on 2022/10/12.
+//
+
+import UIKit
+
+
+private enum Size {
+    static let sampleImageSize = 80.0
+    static let defaultOffset = 20.0
+    static let secondaryOffset = 16
+}
+
+final class ShopBasketCollectionViewCell: BaseCollectionViewCell {
+
+
+    // MARK: - Properties
+    // Cell consists of checkBox,sampleImageView, verticalStackView
+
+
+    private let checkBox: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(systemName: "square.fill"), for: .normal)
+        button.imageView?.tintColor = .systemGray3
+        return button
+    }()
+
+    private let sampleImageView: UIImageView =  {
+        let imageView = UIImageView()
+        imageView.clipsToBounds = true
+        imageView.image = UIImage(named: "WhiteMarble")
+        return imageView
+    }()
+
+    // verticalStackView consists of firstHorizontalStackView, matNameLabel, thicknessLabel,samplePriceLabel
+    private let verticalStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.alignment = .leading
+        stackView.distribution = .fillProportionally
+        stackView.spacing = 1.33
+        return stackView
+    }()
+
+    // firstHorizontalStackView consists of makerLabel,deleteButton
+    private let firstHorizontalStackView: UIStackView = {
```

Comment:
p5.

```suggestion
    private let horizontalStackView: UIStackView = {
```

HStack이 하나만 존재해서 first를 명시하지 않아도 될 거 같아요